src/main/java/transforms/HashTransform.java

```diff
@@ -51,8 +52,8 @@ public HashTransform(
             @JsonProperty( Constants.COLUMNS ) List<String> columns,
             @JsonProperty( Constants.HASH_FUNCTION ) String hashFunction ) {
         this.columns = columns;
-        this.hashFunction = hashFunction;
-        switch ( hashFunction ) {
+        this.hashFunction = StringUtils.isBlank( hashFunction ) ? "sha256" : hashFunction;
```

Comment:
Might be cleaner to create an enum for hash function types, and then if an invalid hash function string is passed, an error will be thrown. You can also make the hashFunction param an Optional, and then if it's not passed you can set a default value.

Reply:
done !